components/suggest/src/store.rs

```diff
@@ -1664,6 +1705,64 @@ mod tests {
                 ]
                 "#]],
             ),
+            (
+                "keyword = `soft l`; non-sponsored only",
+                SuggestionQuery {
+                    keyword: "soft l".into(),
```

Comment:
Oh, I'm so sorry—I meant, let's keep both `soft` and `soft l`, not change it—just to make sure that we handle empty suffixes for low-confidence matches correctly.